src/main/StreamTabs.js

```diff
@@ -15,34 +15,45 @@ const styles = StyleSheet.create({
   },
 });
 
-export default TabNavigator(
-  {
-    subscribed: {
-      screen: props => <SubscriptionsContainer {...props.screenProps} />,
-      navigationOptions: {
-        tabBarLabel: props => (
-          <Text style={[styles.tab, { color: props.tintColor }]}>
-            <FormattedMessage id="Subscribed" defaultMessage="Subscribed" />
-          </Text>
-        ),
-      },
-    },
-    streams: {
-      screen: props => <StreamListContainer {...props.screenProps} />,
-      navigationOptions: {
-        tabBarLabel: props => (
-          <Text style={[styles.tab, { color: props.tintColor }]}>
-            <FormattedMessage id="All streams" defaultMessage="All streams" />
-          </Text>
-        ),
+type Props = {
+  theme: string,
+};
+
+export default class DynamicTabBar extends Component<Props> {
```

Comment:
Rename `DynamicTabBar` to more clearer name.

Reply:
Renamed both classes to better reflect their positions.